tests/integration/targets/meraki_device/tasks/main.yml

```diff
@@ -1,246 +1,265 @@
 ---
-- block:
+- name: Block of tests
+  block:
   # This is commented out because a device cannot be unclaimed via API
   # - name: Claim a device into an organization
-  #   meraki_device:
-  #     auth_key: '{{auth_key}}'
-  #     org_name: '{{test_org_name}}'
-  #     serial: '{{serial}}'
+  #   cisco.meraki.meraki_device:
+  #     auth_key: '{{ auth_key }}'
+  #     org_name: '{{ test_org_name }}'
+  #     serial: '{{ serial }}'
   #     state: present
   #   delegate_to: localhost
   #   register: claim_device_org
 
-  # - assert:
+  # - ansible.builtin.assert:
   #     that:
   #       - claim_device_org.changed == true
 
   - name: Create network
-    meraki_network:
-      auth_key: '{{auth_key}}'
-      org_name: '{{test_org_name}}'
-      net_name: '{{test_net_name}}'
+    cisco.meraki.merak_network:
```

Comment:
This is a typo so the integration test didn't pass. Please resolve and I can run the tests.

Reply:
I pushed it. So you can test it ;)